capture-sdk/sdk/src/doc/source/features.rst

```diff
@@ -421,6 +421,20 @@ You can show customized activity indicator on this screen. You can pass your cus
             .setLoadingIndicatorAdapter(customLottiLoadingIndicatorAdapter)
             .build();
 
+Error Screen Customization
+--------------------------
+
+You can show custom back navigation button on bottom navigation bar. You can pass your custom ``ErrorNavigationBarBottomAdapter`` implementation to
+``GiniCapture.Builder``:
+
+.. code-block:: java
+
+    CustomErrorNavigationBarBottomAdapter customErrorNavigationBarBottomAdapter = new CustomErrorNavigationBarBottomAdapter();
+
+    GiniCapture.newInstance()
+            .setErrorNavigationBarBottomAdapter(customErrorNavigationBarBottomAdapter)
+            .build();
+
```

Comment:
[Request] We should also mention that they should handle the `CameraActivity.RESULT_ENTER_MANUALLY` result code which is returned by the SDK when the user clicks the "Enter manually" button on the error screen.